src/lib/input/input-container.ts

```diff
@@ -356,10 +357,7 @@ export class MdInputContainer implements AfterViewInit, AfterContentInit {
     @Optional() private _parentFormGroup: FormGroupDirective) { }
 
   ngAfterContentInit() {
-    if (!this._mdInputChild) {
-      throw new MdInputContainerMissingMdInputError();
-    }
-
+    this._validateInputChild();
```

Comment:
[They're being used in the view.](https://github.com/angular/material2/blob/master/src/lib/input/input-container.html#L3)